smarts/p3d/renderer.py

```diff
@@ -55,25 +55,16 @@
     loadPrcFileData,
 )
 
-from . import glsl, models
-from .colors import Colors, SceneColors
-from .coordinates import Pose
-from .masks import RenderMasks
-from .scenario import Scenario
+from smarts.core import glsl, models
+from smarts.core.colors import Colors, SceneColors
+from smarts.core.coordinates import Pose
+from smarts.core.masks import RenderMasks
+from smarts.core.renderer_base import DEBUG_MODE, RendererBase
+from smarts.core.scenario import Scenario
 
 # pytype: enable=import-error
```

Comment:
Why is this `# pytype: enable=import-error` here? Perhaps it can be removed.

Reply:
It used to be related to the Panda3d imports.